acknowledgements.tex

```diff
@@ -2,3 +2,9 @@
 Sandia is a multiprogram laboratory operated by Sandia Corporation, a Lockheed Martin Company, for the United States Department of Energy's National Nuclear Security Administration under Contract DE-AC04-94AL85000.
 
 The Google Summer of Code, an international annual program, in which Google awards stipends to all students who successfully complete a requested free and open-source software coding project during the summer.
+
+The author of this paper Francesco Bonazzi thanks the Deutsche
```

Comment:
Well, they are paying me. I'm supposed to thank them (I have used working hours for SymPy).